api/db-cdr/generate-cdr/validate-prerequisites-exist.sh

```diff
@@ -7,14 +7,12 @@ export BQ_PROJECT=$1        # CDR project
 export BQ_DATASET=$2        # CDR dataset
 export CDR_VERSION=$3       # CDR version
 
-PREP_TABLE_RUN="!_prep_tables_!"
```

Comment:
This was something Jon used for make-prep-tables-from-csv which is no longer needed.